bigquery/google/cloud/bigquery/table.py

```diff
@@ -783,22 +783,38 @@ def __init__(self, values, field_to_index):
         self._xxx_field_to_index = field_to_index
 
     def values(self):
+        """Return the values included in this row.
+
```

Comment:
Worth noting that it's not a copy? Also won't it always be a `tuple`? If not, returning it is a "bad idea"

Reply:
It's _supposed_ to always be a tuple. Agreed that it's a little scary, though.

The values themselves can sometimes be mutable, such as with RECORD/STRUCT types. I'll do a deep copy.